company/beliefs_and_values.md

```diff
@@ -9,3 +10,37 @@
 * We believe that nearly all organisations can move faster than they do. We try to help them to do this.
 
 * We believe the adoption of Agile is in it's infancy and that Agile principles (and various spin offs) will underpin all high-performance businesses in years to come.
+
+###Values
+
+* We take care of the our space and those in it.
+
+* We are willing and open to continuous improvement in our culture
+
+* We take care of those around us and those we talk to.
+
+* No one should feel uncomfortable
+
+* Everyone should feel productive
+
+* Be aware of ego
+
+* Actions over words
+
+* Don't be submissive
+
+* Know your audience
+
+* Awareness of culture
+
+* Adapt and iterate culture
+
+* Seek diversity
+
+* Be responsible for our space
+
+* Scout attitude to the office
```

Comment:
Just a general point; I think we should use the term 'being proactive' rather than scouting, as I'm not sure it's as clear.